docs/generated/settings/settings-for-tenants.txt

```diff
@@ -401,9 +401,9 @@ storage.delete_compaction_excise.enabled	boolean	true	set to false to direct Peb
 storage.ingestion.value_blocks.enabled	boolean	true	set to true to enable writing of value blocks in ingestion sstables	application
 storage.max_sync_duration	duration	20s	maximum duration for disk operations; any operations that take longer than this setting trigger a warning log entry or process crash	system-visible
 storage.max_sync_duration.fatal.enabled	boolean	true	if true, fatal the process when a disk operation exceeds storage.max_sync_duration	application
-storage.sstable.compression_algorithm	enumeration	minlz	determines the compression algorithm to use when compressing sstable data blocks for use in a Pebble store; [snappy = 1, zstd = 2, none = 3, minlz = 4]	system-visible
-storage.sstable.compression_algorithm_backup_storage	enumeration	minlz	determines the compression algorithm to use when compressing sstable data blocks for backup row data storage; [snappy = 1, zstd = 2, none = 3, minlz = 4]	system-visible
-storage.sstable.compression_algorithm_backup_transport	enumeration	minlz	determines the compression algorithm to use when compressing sstable data blocks for backup transport; [snappy = 1, zstd = 2, none = 3, minlz = 4]	system-visible
+storage.sstable.compression_algorithm	enumeration	snappy	determines the compression algorithm to use when compressing sstable data blocks for use in a Pebble store; [snappy = 1, zstd = 2, none = 3, minlz = 4]	system-visible
```

Comment:
what's with the `minlz` -> `snappy` changes in some of these files?

Reply:
It looks like its because of this change, which makes local `./dev gen` generate this diff. Its causing failures in builds now because the docs generated on mac are different than those on linux. It looks like rafi already brought it up in #storage